sklearn/tree/export.py

```diff
@@ -415,3 +416,108 @@ def recurse(tree, node_id, criterion, parent=None, depth=0):
+        Only relevant for classification and not supported for multi-output.
+        If ``True``, shows a symbolic representation of the class name.
+
+    label : {'all', 'root', 'none'}, optional (default='all')
+        Whether to show informative labels for impurity, etc.
+        Options include 'all' to show at every node, 'root' to show only at
+        the top root node, or 'none' to not show at any node.
+
+    filled : bool, optional (default=False)
+        When set to ``True``, paint nodes to indicate majority class for
+        classification, extremity of values for regression, or purity of node
+        for multi-output.
+
+    leaves_parallel : bool, optional (default=False)
+        When set to ``True``, draw all leaf nodes at the bottom of the tree.
+
+    impurity : bool, optional (default=True)
+        When set to ``True``, show the impurity at each node.
+
+    node_ids : bool, optional (default=False)
+        When set to ``True``, show the ID number on each node.
+
+    proportion : bool, optional (default=False)
+        When set to ``True``, change the display of 'values' and/or 'samples'
+        to be proportions and percentages respectively.
+
+    rotate : bool, optional (default=False)
+        When set to ``True``, orient tree left to right rather than top-down.
+
+    rounded : bool, optional (default=False)
+        When set to ``True``, draw node boxes with rounded corners and use
+        Helvetica fonts instead of Times-Roman.
+
+    special_characters : bool, optional (default=False)
+        When set to ``False``, ignore special characters for PostScript
+        compatibility.
+
+    out : {'matplotlib', 'jupyter'}, optional (default='matplotlib')
+        Options include 'matplotlib' to plot using matplotlib library,
+        'jupyter' to plot to Jupyter Notebook environment.
+    """
+    if out not in ('matplotlib', 'jupyter'):
+        raise ValueError(
+            "Incorrect 'out' value passed (%s). "
+            "Use 'jupyter' or 'matplotlib'." % out)
+    try:
+        import graphviz
+    except ImportError:
+        raise ImportError('Plotting requires graphviz to be installed.')
+
+    in_memory_dot_file = six.StringIO()
+    export_graphviz(
+        decision_tree, out_file=in_memory_dot_file, max_depth=max_depth,
+        feature_names=feature_names, class_names=class_names, label=label,
+        filled=filled, leaves_parallel=leaves_parallel, impurity=impurity,
+        node_ids=node_ids, proportion=proportion, rotate=rotate,
+        rounded=rounded, special_characters=special_characters)
+    src = graphviz.Source(in_memory_dot_file.getvalue())
+    image = src.pipe(format='png')
+    if out == 'jupyter':
+        try:
+            from IPython.display import Image
+        except ImportError:
+            raise ImportError(
+                'Plotting to Jupyter Notebook requires Jupyter Notebook.')
+        return Image(image, height=height, width=width)
```

Comment:
In fact, they will have multiple solution to manage height/width with their favorite library.